src/components/UI/Navigation/Hamburger.js

```diff
@@ -0,0 +1,79 @@
+import styled from '@emotion/styled';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const HamburgerDiv = styled.div`
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+
+  div {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    height: 100%;
+  }
+
+  /* Hamburger bars */
+  .first,
+  .second,
+  .third {
+    background: ${(props) => props.theme.colors.white3};
+    height: 2px;
+    margin: 2px 2px;
+    border-radius: 3px;
+    transition: all 0.3s;
+  }
+
+  .second {
+    margin: 2px 1px;
+  }
+
+  &.open {
+    .first,
+    .third {
+      width: 17.5px;
+    }
+    .first {
+      transform: rotate(35deg) translate(2.3px, 4.9px);
```

Comment:
![image](https://user-images.githubusercontent.com/13508038/78131745-dda8fd00-741b-11ea-881e-184d6a2e7b54.png)

Currently when the hamburger becomes an `X`, the `X` is not centered anymore into the button.

This worked for me:
```suggestion
      transform: translate3d(0, 6px, 0) rotate(35deg);
      transform-origin: center;
```
(using `translate3d` because it is GPU accelerated)

and then for the `.third`:
```css
transform: translate3d(0, -6px, 0) rotate(-35deg);
transform-origin: center;
```